pom.xml

```diff
@@ -107,6 +150,11 @@
             <artifactId>bigqueue</artifactId>
             <version>1.7.0.4</version>
         </dependency>
+        <dependency>
```

Comment:
If you are not directly using it, it's best practice not to. It will come as a transitive dependency from Big Queue